<ui version="4.0" >
 <class>MarbleControlBox</class>
 <widget class="QWidget" name="MarbleControlBox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>185</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>410</height>
      </size>
     </property>
     <property name="whatsThis" >
      <string>This control panel allows you to adjust the map according to your needs.</string>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="toolBoxTab1" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>169</width>
        <height>497</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Navigation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="5" column="0" >
          <widget class="Line" name="line_2_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2_2" >
           <property name="text" >
            <string>&amp;Search</string>
           </property>
           <property name="buddy" >
            <cstring>searchLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLineEdit" name="searchLineEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Search line</string>
           </property>
           <property name="whatsThis" >
            <string>Here you can enter the name of the location you are searching for.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="SearchListView" name="locationListView" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Location Browser</string>
           </property>
           <property name="whatsThis" >
            <string>Double Click an entry of this list to get it displayed on the map.</string>
           </property>
           <property name="horizontalScrollBarPolicy" >
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors" >
            <bool>false</bool>
           </property>
           <property name="iconSize" >
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="layoutMode" >
            <enum>QListView::Batched</enum>
           </property>
           <property name="viewMode" >
            <enum>QListView::ListMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <property name="horizontalSpacing" >
            <number>6</number>
           </property>
           <property name="verticalSpacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QToolButton" name="moveLeftButton" >
             <property name="toolTip" >
              <string>Left</string>
             </property>
             <property name="whatsThis" >
              <string>Rotates the earth counterclockwise around its axis.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-left.png</iconset>
             </property>
             <property name="shortcut" >
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <widget class="QToolButton" name="zoomOutButton" >
             <property name="toolTip" >
              <string>Zoom Out</string>
             </property>
             <property name="whatsThis" >
              <string>Zoom out by pressing this button to see less details.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/zoom-out.png</iconset>
             </property>
             <property name="shortcut" >
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QToolButton" name="moveUpButton" >
             <property name="toolTip" >
              <string>Up</string>
             </property>
             <property name="whatsThis" >
              <string>Tilts the earth's axis towards the user.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-up.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QToolButton" name="goHomeButton" >
             <property name="toolTip" >
              <string>Reset View</string>
             </property>
             <property name="whatsThis" >
              <string>Click this button to restore the original zoom level and view angle.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/go-home.png</iconset>
             </property>
             <property name="shortcut" >
              <string>Home</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QToolButton" name="moveDownButton" >
             <property name="toolTip" >
              <string>Down</string>
             </property>
             <property name="whatsThis" >
              <string>Tilts the earth's axis away from the user.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-down.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QToolButton" name="zoomInButton" >
             <property name="toolTip" >
              <string>Zoom In</string>
             </property>
             <property name="whatsThis" >
              <string>Zoom in by pressing this button to see more details.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/zoom-in.png</iconset>
             </property>
             <property name="shortcut" >
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QSlider" name="zoomSlider" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="MinimumExpanding" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip" >
              <string>Zoom Slider</string>
             </property>
             <property name="whatsThis" >
              <string>Use this slider to adjust the zoom level of the map.</string>
             </property>
             <property name="minimum" >
              <number>1000</number>
             </property>
             <property name="maximum" >
              <number>2500</number>
             </property>
             <property name="value" >
              <number>1050</number>
             </property>
             <property name="sliderPosition" >
              <number>1050</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="tickPosition" >
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval" >
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1" >
            <widget class="QFrame" name="m_pSpacerFrame" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>16</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QToolButton" name="moveRightButton" >
             <property name="toolTip" >
              <string>Right</string>
             </property>
             <property name="whatsThis" >
              <string>Rotates the earth clockwise around its axis.</string>
             </property>
             <property name="text" >
              <string>...</string>
             </property>
             <property name="icon" >
              <iconset resource="marble.qrc" >:/icons/arrow-right.png</iconset>
             </property>
             <property name="shortcut" >
              <string>Right</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="legendTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>112</width>
        <height>112</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Legend</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="LegendBrowser" name="legendBrowser" >
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolBoxTab2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>112</width>
        <height>112</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Map View</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="KAtlasThemeSelectView" name="katlasThemeSelectView" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Map View Browser</string>
         </property>
         <property name="whatsThis" >
          <string>Here you can choose the preferred map view from different topics.</string>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gpsOption" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>185</width>
        <height>440</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Gps Option</string>
      </attribute>
      <widget class="QWidget" name="" >
       <property name="geometry" >
        <rect>
         <x>12</x>
         <y>12</y>
         <width>151</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="m_latLabel" >
                <property name="text" >
                 <string>Latitude</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="m_latSpin" >
                <property name="minimumSize" >
                 <size>
                  <width>82</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="decimals" >
                 <number>4</number>
                </property>
                <property name="maximum" >
                 <double>90.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>16</width>
                <height>49</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QRadioButton" name="m_nRadio" >
                <property name="text" >
                 <string>N</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_sRadio" >
                <property name="text" >
                 <string>S</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="m_lonLabel" >
                <property name="text" >
                 <string>Longitude</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="m_lonSpin" >
                <property name="minimumSize" >
                 <size>
                  <width>82</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="decimals" >
                 <number>4</number>
                </property>
                <property name="maximum" >
                 <double>180.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>16</width>
                <height>49</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QRadioButton" name="m_eRadio" >
                <property name="text" >
                 <string>E</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="m_wRadio" >
                <property name="text" >
                 <string>W</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="m_trackGps" >
              <property name="text" >
               <string>Track Gps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="m_catchGps" >
              <property name="text" >
               <string>Catch Gps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="m_gpsDrawBox" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Draw</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_gpsGoButton" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Go To</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchListView</class>
   <extends>QListView</extends>
   <header>lib/searchlistview.h</header>
  </customwidget>
  <customwidget>
   <class>LegendBrowser</class>
   <extends>QTextBrowser</extends>
   <header>lib/LegendBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>KAtlasThemeSelectView</class>
   <extends>QListView</extends>
   <header>lib/katlasthemeselectview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="marble.qrc" />
 </resources>
 <connections/>
</ui>
